src/components/action-list/ActionList.jsx

```diff
@@ -15,10 +27,19 @@ export const ALIGNMENT = {
   STRETCH: 'stretch'
 };
 
-const ActionList = ({children, toTop, direction, align, noWrap, className, ...props}) => {
+type ActionListPropsType = {
+  children: Node,
+  toTop?: ?boolean,
```

Comment:
is the problem with optional type `?: ?` in sg itself or somewhere else?

Reply:
in our main project, we use different patterns, some of them can results to `null`, some of them to `undefined`, but at the end it doesn't matter because `classnames` just doesn't use `null`\`undefined`.

Reply:
I see 👍